scico/_autograd.py

```diff
@@ -69,43 +56,21 @@ def conjugated_grad(*args, **kwargs):
     return conjugated_grad_aux if has_aux else conjugated_grad
 
 
+# Append docstring from original jax function
+grad.__doc__ = _append_jax_docs(grad)
```

Comment:
These docs aren't rendering correctly (e.g. see the text after Args: in https://scico--119.org.readthedocs.build/en/119/_autosummary/scico.html#scico.grad). I believe I got something similar working in `scico.random` after a lot of fighting, see https://github.com/lanl/scico/blob/76ad408e825c1549df4a26a2c96dd87fb95acefb/scico/random.py#L106

Reply:
Strange, I thought it rendered correctly when I tried to build docs locally. But I guess I missed that. Thanks for the suggestion; I'll give it a try.

Reply:
Not so simple, unfortunately.

Reply:
Fixed in 376c06a.